[stable][selenium] Add the option of adding nodeSelectors

gezb opened the pull request:
These can be applied globally or per Selenium component (hub, chrome, chromeDebug, firefox or firefoxDebug) 

*****************************************************************************************

Thank you for contributing to kubernetes/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the 
history. This will make it easier to identify new changes. The PR will be squashed 
anyways when it is merged. Thanks.

*****************************************************************************************

gezb committed:
[stable][selenium] Add the option of adding nodeSelectors either globally or per component

gezb commented:
/hold

Found some issues with this I will push a fix once I have sorted it

gezb commented:
/unhold 

This turned out to be the way I was using labels

gezb commented:
I have fixed a few issues I found, Let me know if yow want me to rebase and squash the changes into a single commit

jdumars commented:
/ok-to-test

timstoop commented on stable/selenium/Chart.yaml:
```diff
@@ -1,5 +1,5 @@
 name: selenium
-version: 0.2.6
+version: 0.2.7
```
This adds a new feature, so please bump the minor version.

timstoop reviewed (changes requested):
Can you please update the README as well?

gezb commented:
@timstoop I have rebased against master, updated the README.md and corrected the version then squashed all the changes into a single commit

timstoop commented:
No need to squash, they will be squashed when merged and for reviewers it's convenient to have all the changes serially applied instead. I'll take a look, though.

timstoop commented:
/lgtm

unguiculus commented:
The way you implemented this, a global node selector takes precendence over individual ones. I would actually expect it the other way around. Any reason why you implemented it this way?

unguiculus commented:
/assign

gezb committed:
[stable][selenium] Change precendence of global NodeSelector values

gezb commented:
@unguiculus I think nodeselectors for the individual items was a afterthought so thats why global  took precedence but it makes sense what you say I have changed this and updated the Readme to make this clear

unguiculus commented:
/lgtm
